include/deal.II/distributed/fullydistributed_tria.h

```diff
@@ -0,0 +1,237 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2008 - 2019 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii_fullydistributed_tria_h
+#define dealii_fullydistributed_tria_h
```

Comment:
We generally use `CamelCase` for class names and `under_scores` for variable and file names. Can you rename the file to `full_distributed_tria.h` and adjust the `#include` guard here?

Reply:
I have done it locally. But did you remove the y intentionally?